bc_obps/service/tests/test_contact_service_v2.py

```diff
@@ -0,0 +1,33 @@
+from registration.schema.v1.contact import ContactFilterSchema
+import pytest
+from service.contact_service_v2 import ContactServiceV2
+from model_bakery import baker
+
+pytestmark = pytest.mark.django_db
+
+
+class TestListContactService:
+    @staticmethod
+    def test_list_contacts():
+
+        user = baker.make_recipe('utils.cas_admin')
+        contact1 = baker.make_recipe('utils.contact')
+        contact2 = baker.make_recipe('utils.contact')
+
+        operator1 = baker.make_recipe('utils.operator')
+
+        operator2a = baker.make_recipe('utils.operator')
+        operator2b = baker.make_recipe('utils.operator')
+
+        # contact 1 is associated with one operator, count = 1
+        operator1.contacts.set([contact1])
+
+        # contact 2 belongs to two operators count = 3
+        operator2a.contacts.set([contact2])
+        operator2b.contacts.set([contact2])
```

Comment:
hmmm this has raised a red flag for me (I know that this isn't something you introduced, turns out we've had our db structured this way for a long time). I don't think our app should allow for the same contact to be assigned to multiple operators - that could mean that Operator2a modifies/deletes contact data and then Operator2b either sees updated contact data or loses data.

we've got M2M fields configured on the Operator model for both contacts and documents, when they should IMO be Many-to-1 relationship. One operator shouldn't ever be able to access another operator's documents or contacts. Do you agree?

Reply:
Yes, I agree. I can't think of why documents was m2m; that just seems like a mistake. For contacts, I think we might have originally set them up as m2m because we thought a consultant might work with multiple operators. But I don't think that makes sense with how the app works anymore.

Reply:
created tech debt ticket https://github.com/bcgov/cas-registration/issues/2680